src/plugins/ccode/coder.hpp

```diff
@@ -0,0 +1,55 @@
+/**
+ * @file
+ *
+ * @brief Definition of string encoding and decoding class
+ *
+ * @copyright BSD License (see LICENSE.md or https://www.libelektra.org)
+ *
+ */
+
+#ifndef ELEKTRA_CODER_HPP
+#define ELEKTRA_CODER_HPP
+
+#include <vector>
+
+#include <kdblogger.h>
+
+#include <keyset.hpp>
+
+using CppKeySet = kdb::KeySet;
+using CppKey = kdb::Key;
+
+using std::string;
```

Comment:
This should not be done in header files.

Reply:
I move the code into the namespace `elektra` in one fo the latest commits. Hopefully that works for you.